src/panels/lovelace/cards/hui-picture-elements-card.js

```diff
@@ -36,8 +33,7 @@ class HuiPictureElementsCard extends LocalizeMixin(EventsMixin(PolymerElement))
         transform: translate(-50%, -50%);
       }
       div.element {
-        padding: 4px;
-        line-height: 1em;
+        padding: 8px;
```

Comment:
Why would elements need padding? They are icons, buttons etc? If it's to be a touch target for the icons, well than we should only apply that to the icon.

Reply:
it's for state-text only
![unbenannt](https://user-images.githubusercontent.com/11984118/41983196-ab1cb6e0-7a2d-11e8-9b9f-742838a3e21d.png)

Reply:
Then state text should have it applied, not the generic element class.

Reply:
`div.element` is only state-text

Reply:
... OK, I create a new class to avoid confusion